tools/depends/target/Toolchain.cmake.in

```diff
@@ -106,6 +106,8 @@ set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 set(CMAKE_FIND_FRAMEWORK LAST)
 set(ENV{PKG_CONFIG_LIBDIR} @prefix@/@deps_dir@/lib/pkgconfig:@prefix@/@deps_dir@/share/pkgconfig)
+# set configured FFmpeg configure options
+set(FFMPEG_CONFIGURE_OPTIONS "@ffmpeg_options@")
```

Comment:
has to be moved to `tools/depends/Makefile.include.in`

Reply:
done